macos/sol-macOS/lib/ApplicationSearcher.swift

```diff
@@ -52,12 +52,14 @@ class ApplicationSearcher: NSObject {
         let resourceKeys: [URLResourceKey] = [
           .isExecutableKey,
           .isApplicationKey,
+          .isAliasFileKey
```

Comment:
I just realized that this change is probably not necessary and a remnant from testing. I will remove it tomorrow.